Detector/DetFCChhHCalTile/compact/FCChh_HCalBarrel_TileCal.xml

```diff
@@ -2,30 +2,34 @@
 <lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
   xmlns:xs="http://www.w3.org/2001/XMLSchema"
   xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
-
   <info name="FCChh_HCalBarrel_TileCal"
-    title="Barrel Calorimeters"
-    author="C. Solans, J. Lingemann"
+    title="Barrel Calorimeter"
+    author="C. Neubueser"
     url="no"
     status="development"
     version="1.0">
-    <comment>HCal layout based on ATLAS HCal</comment>
+    <comment>HCal layout based on ATLAS HCal, with realistic longitudinal segmentation and steel support</comment>
   </info>
-  
   <define>
-    <!-- 20cm for support is reduced to 18cm, 1cm is needed to place max number of trapazoids and 1cm needed to compensate for faceplate-->
-    <constant name="BarHCal_rmaxWithBuffer" value="BarHCal_rmax-180*mm"/>
+    <!-- 21.2cm for support, 1cm is needed to place max number of trapazoids and 1cm needed to compensate for faceplate-->
+    <constant name="BarHCal_rmaxWithBuffer" value="BarHCal_rmax-212*mm"/>
```

Comment:
Sorry, now I'm completely confused. What did you actually delete? 

My original comment was just to change this to use the variable defined below instead of explicitly writing the thickness again. I.e. something like this snippet:

```xml
<constant name="BarHCal_steel_support_thickness" value="212*mm" />
<constant name="BarHCal_rmaxWithBuffer" value="BarHCal_rmax-BarHCal_steel_support_thickness"/>
```